src/services/WeatherService.js

```diff
@@ -1,11 +1,33 @@
-import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react'
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
 
-export const weatherApi = createApi({
-    reducerPath: 'weatherApi',
-    baseQuery: fetchBaseQuery({ baseUrl: '' }),
-    endpoints: (builder) => ({
-      getPokemonByName: builder.query<Pokemon, string>({
-        query: (name) => `pokemon/${name}`,
-      }),
+const apikey = "w74RorP4hf7wdUwIUlKNPhRxIa4fEVjW";
+
+export const WeatherApi = createApi({
+  reducerPath: "WeatherApi",
+  baseQuery: fetchBaseQuery({ baseUrl: "http://dataservice.accuweather.com" }),

```

Comment:
Same for base url, move it to constants?

Reply:
ok